lib/components/app/batch-routing-panel.tsx

```diff
@@ -33,6 +61,14 @@ class BatchRoutingPanel extends Component<Props> {
     this.setState({ planTripClicked: true })
   }
 
+  handleOpenAdvanceSettings = () => {
+    this.setState({ showAdvancedModeSettings: true })
+  }
+
+  handleCloseAdvanceSettings = () => {
+    this.setState({ showAdvancedModeSettings: false })
+  }
```

Comment:
nitpick: refactor this to be a single method that takes true/false as an argument?

Reply:
Good catch, I've gone one step further and eliminated the methods entirely